backend/src/util/utilities.ts

```diff
@@ -14,3 +14,7 @@ export const decimalSeparatorByLanguage = (a: Decimal, language: string): string
   i18n.setLocale(rememberLocaleToRestore)
   return result
 }
+
+export const fullName = (firstName: string, lastName: string): string => {
+  return [firstName, lastName].filter(Boolean).join(' ')
+}
```

Comment:
This is just to make it even shorter - you can ignore this suggestion if you want
```suggestion
export const fullName = (firstName: string, lastName: string): string =>
  [firstName, lastName].filter(Boolean).join(' ')
```